core/db/migrate/20180404215857_add_default_state_to_payment.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddDefaultStateToPayment < ActiveRecord::Migration[5.1]
+  def change
+    change_column_default(:spree_payments, :state, Spree::Payment::CHECKOUT)
```

Comment:
We should use the value of the constant here so we do not rely on the code may change in the future.